content/en/docs/advanced/creating_custom_cloud_images.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Creating Custom Cloud Images"
+linkTitle: "Creating Custom Cloud Images"
+weight: 1
+description: A comprehensive guide to creating custom cloud images with Kairos using the latest tools
+---
+
+This guide provides a complete walkthrough for creating custom cloud images with Kairos. It covers the entire process from start to finish, using the latest tools like `kairos-init` and `AuroraBoot`.
+
+## Overview
+
+Kairos provides several tools to create custom cloud images:
+
+1. **kairos-init**: A tool for creating base container images
+2. **AuroraBoot**: A tool for generating bootable images (ISOs, cloud images, etc.)
+3. **Customization tools**: Various methods to customize the images
+
+## Prerequisites
+
+Before starting, ensure you have:
+
+- Docker installed and running
+- A Linux machine with KVM support (for testing)
+- Basic understanding of container images and cloud configurations
+
+## Step 1: Creating a Base Image with kairos-init
+
+The first step is to create a base container image using [`kairos-init`](https://github.com/kairos-io/kairos-init). This tool allows you to create a custom base image from popular distributions.
+
+Here's a basic example of creating a base image:
+
+```bash
+docker build -t my-custom-image - <<EOF
+ARG BASE_IMAGE=ubuntu:24.04
+
+FROM quay.io/kairos/kairos-init:v0.4.3 AS kairos-init
```

Comment:
0.4.3 or maybe even better lets take this out into a var thats gets updated by renovate like others around?

Reply:
good point. Fixed